spring-boot-tools/spring-boot-configuration-metadata/src/test/resources/metadata/configuration-metadata-emptygroup.json

```diff
@@ -0,0 +1,18 @@
+{
+  "groups": [
+    {
+      "name": "",
```

Comment:
I am not sure that this is legit actually. It should be re-attached to the "root" group somehow. I am assuming that you end up in this situation with a plain `@ConfigurationProperties` annotation on your bean?